src/playwright/playwright.config.ts

```diff
@@ -37,6 +32,46 @@ export default defineConfig({
       // developers feedback quickly.
       timeout: 1000,
     },
+    ...[
+      {
+        device: 'Desktop Chrome',
+
+        // - https://playwright.dev/docs/browsers#chromium-new-headless-mode
+        // - https://developer.chrome.com/blog/chrome-headless-shell
+        channel: 'chromium',
+      },
+      { device: 'Desktop Firefox' },
+      { device: 'Desktop Safari' },
```

Comment:
There's unfortunately no way to tag tests with the browser name so it shows up in the test report. But you [can](https://glide-core.crowdstrike-ux.workers.dev/finish-configuring-playwright/test-report#?q=annot%3Awebkit) filter the report using `annot:[browser]`.